calculator/src/Calculator_function.js

```diff
@@ -0,0 +1,73 @@
+import React,{useState} from 'react';
+
+function Calculator_function(){
```

Comment:
```suggestion
function CalculatorFunction(){
```
在React里 组件名称推荐使用`PascalCase` like `FunctionalCalculator`

Reply:
另外 文件名一般使用 `CalculatorFunction.jsx`跟组件一致 或者 `calculator-function.jsx`

不同项目会有不同要求 但是同一项目内要做到命名规则一致